sonarqube/content.md

```diff
@@ -6,11 +6,26 @@
 
 # How to use this image
 
-This Docker image contains the Community Edition of SonarQube.
+Here you'll find the Docker image for the Community Edition of SonarQube and preview versions of the Docker images for Developer Edition and Enterprise Edition.
 
-## Run SonarQube
+## Docker Host Requirements
 
-The server is started this way:
+Because SonarQube uses an embedded Elasticsearch, make sure that your Docker host configuration complies with the [Elasticsearch production mode requirements](https://www.elastic.co/guide/en/elasticsearch/reference/current/docker.html#docker-cli-run-prod-mode) and [File Descriptors configuration](https://www.elastic.co/guide/en/elasticsearch/reference/current/file-descriptors.html).
+
+For example, on Linux, you can set the recommended values for the current session by running the following commands as root on the host:
+
+```console
+sysctl -w vm.max_map_count=262144
+sysctl -w fs.file-max=65536
+ulimit -n 65536
+ulimit -u 4096
+```
+
+## Get Started in Two Minutes Guide
+
+/!\  This section shows you how to install a demo instance. When you are ready to move to production, take some time to read the Configuration section below.
+
+Start the server by running:
 
 ```console
 $ docker run -d --name sonarqube -p 9000:9000 %%IMAGE%%
```

Comment:
We use `%%IMAGE%%` here and in one of the last sections.
Should we use it everywhere else where we use docker commands?

Reply:
yes we should